integration-tests/http/__tests__/cart/store/cart.spec.ts

```diff
@@ -1922,7 +1922,7 @@ medusaIntegrationTestRunner({
               expect(res.data.order).toEqual(successData)
             }
 
-            expect(failure.length).toBeGreaterThan(0)
+            expect(failure.length).toBe(0)
```

Comment:
This does not fail anymore, and it is expected. As all the runs should respect the lock, and only the first will complete the cart and create the order. All the rest will just return the order created.